common/src/main/java/com/wynntils/models/wynnfont/WynnFontModel.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright © Wynntils 2025.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.wynnfont;
+
+import com.wynntils.core.components.Model;
+import com.wynntils.utils.colors.CustomColor;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+public class WynnFontModel extends Model {
+    private static final char NEGATIVE_SPACE = '\uE012';
+    private static final char NEGATIVE_SPACE_EDGE = '\u2064';
+    private static final char BACKGROUND = '\uE00F';
+    private static final List<Character> normalCharacters = List.of(
+            'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u',
+            'v', 'w', 'x', 'y', 'z', '?', '[', ']', '\\', '%', '&', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9');
+    private static final List<Character> fancyCharacters = List.of(
+            '\uE040', '\uE041', '\uE042', '\uE043', '\uE044', '\uE045', '\uE046', '\uE047', '\uE048', '\uE049',
+            '\uE04A', '\uE04B', '\uE04C', '\uE04D', '\uE04E', '\uE04F', '\uE050', '\uE051', '\uE052', '\uE053',
+            '\uE054', '\uE055', '\uE056', '\uE057', '\uE058', '\uE059', '\uE05A', '\uE05B', '\uE05C', '\uE05D',
+            '\uE05E', '\uE05F', '\uE060', '\uE061', '\uE062', '\uE063', '\uE064', '\uE065', '\uE066', '\uE067',
+            '\uE068', '\uE069');
+    private static final Map<Character, Character> normalToFancy = new HashMap<>();
+
+    private void createFontMaps() {
```

Comment:
I think Map.of() would be better here. It'll give us concrete bindings from char->fancy instead of having to rely on index matching